scalafix-rules/src/main/scala/scalafix/internal/rule/RemoveUnused.scala

```diff
@@ -106,9 +106,8 @@ class RemoveUnused(config: RemoveUnusedConfig)
             Patch.replaceTree(v, "_")
           case t @ Pat.Typed(v: Pat.Var, _) if checkUnusedPatTree(t) =>
             Patch.replaceTree(v, "_")
-          case b: Pat.Bind if checkUnusedPatTree(b) =>
-            val removableTokens = b.tokens.dropRightWhile(!_.is[Token.At])
```

Comment:
this was incorrectly picking up the nested bind when attempting to strip the LHS of the outer one